suite/runsuite_wrapper.pl

```diff
@@ -389,6 +389,8 @@
                 );
```

Comment:
Would be good to also mention x86-64 in the PR description and title.

Reply:
These tests have also been seen to fail on aarch64 though.
Aren't references to the issues enough?

Reply:
Well, since this PR is disabling them for a specific suite, it makes sense to mention the suite in the description.

Reply:
Could you point to an aarch64 failure?  Prior to the ptrace fix they failed due to permissions; after that fix they have never failed AFAIK.  I logged into that test machine and ran each test 200x in a row w/ no failures manually as I noted in the issue/PR.

Reply:
> Adds two tests to the unix ignore list:

I'm agreeing with @abhinav92003  -- there is no ignore list for all of "unix": this PR is adding these only to the x86_64 Linux list.

Reply:
Done